decidim-system/spec/system/organizations_spec.rb

```diff
@@ -88,6 +88,29 @@
       end
     end
 
+    describe "edits organization name" do
+      let!(:organization) do
+        create(:organization, name: { ca: "", en: "Citizen Corp", es: "" }, default_locale: :en, available_locales: ["en"], description: { en: "large text" })
+      end
+
+      before do
+        click_on "Organizations"
+        click_on "New"
+      end
+
+      it "creates new organization with incorrect organization admin name" do
+        fill_in "Name", with: "Citizen Corp 2"
+        fill_in "Reference prefix", with: "CCORP"
+        fill_in "Organization admin name", with: "system@system.com"
+
+        click_on "Create organization & invite admin"
```

Comment:
The initial bug report says that you're actually having the error but the Organization is silently created:

> (...) i got an error but the organisation was still created without the admin (...)

So, with the current spec this pass in `develop`

You need to check that the Organization count actually doesn't change. This is done with the following matcher in rspec:

```suggestion
        expect do
          click_on "Create organization & invite admin"
        end.not_to change(Decidim::Organization, :count)
```